src/content/get-started/overview.mdx

```diff
@@ -16,31 +16,37 @@ If you are interested in using Okteto's SaaS offering, you can ignore this page
 
 In order to fully install Okteto, you'll need the following:
```

Comment:
this comment is for the full guide: I think this guide is trying to do too much. Our assumption is that this is a getting started, meant for those who are just learning about okteto and want to try it out. I think there's a lot of redundancy between this doc, the quickstart guides of each cloud provider, and the rest of the docs. 

I think that on this guide we should remove pretty much everything, and just stay with the following:
1. What is Okteto (one paragraph max)
2. How to get a license (start your trial here!)
3. Video on how to install 
4. Link to the quickstart install guides  (the list of cloud-specific providers)
5. IF you want to learn more about how configure and troubleshoot Okteto Self-Hosted go here (with a link to the more in-depth doc section)
For a beginner, we don't want them to install everything from zero. we want them to pick their cloud provider, copy the commands, follow the guide, and have a working instance in less than 1 hour. Anything that doesn't contribute to that (explain the components, troubleshooting , how to uninstall) I think should live somewhere else in our docs.

Reply:
I made another pass, @rberrelleza could you give it another pass?
I have seen other installation guides, and it's quite common to mention troublleshooting an uninstall in the installation overview doc